Dockerfile

```diff
@@ -18,16 +18,13 @@ RUN sed 's/#force_color_prompt=yes/force_color_prompt=yes/' /etc/skel/.bashrc >
 ADD .dockerfiles/.vimrc /root/
 
 WORKDIR /code/data-store
-ADD requirements-dev.txt .
+ADD requirements-dev.txt requirements.txt ./
```

Comment:
From the Dockerfile reference: "If multiple <src> resources are specified, either directly or due to the use of a wildcard, then <dest> must be a directory, and it must end with a slash /."